src/openzaak/components/catalogi/admin/utils.py

```diff
@@ -174,7 +174,12 @@ def import_zaaktype_for_catalogus(
 
     uuid_mapping = {}
 
+    files_not_found = []
+    files_found = []
+
     with zipfile.ZipFile(import_file, "r") as zip_file:
+
+        files_received = zip_file.namelist()
```

Comment:
Now you have both folder names and filenames here. I think it's better to include only files.